### PR TITLE
fixed display:table thing for payment modal

### PR DESCRIPTION
**Pull request by aykamko**

### CODE REVIEW

File: revolv/templates/base/base.html [hunk 1]
```diff
@@ -61,6 +61,19 @@
           reveal: {
               animation: 'fade',
               dismiss_modal_class: 'close-revolv-reveal-modal',
+              css: {
+                  open: {
+                      'opacity': 0,
+                      'visibility': 'visible',
+                      // NOTE: you should manually change display of your modal
+                      // in the scss
```
Comment (noahsark769) [inline revolv/templates/base/base.html]: probably we should explain why, if we're gunna have the comment

**Comment by noahsark769** [comment]
This fixes #171 and #172 only, right?

**Comment by aykamko** [comment]
Yeah, just those two. I'll expand the comment

**Comment by noahsark769** [comment]
kk cool

**Pull request merged**